### DISCUSSION TITLE
constrained versus unconstrained integration

### DISCUSSION THREAD

**Discussion by dimurali93** [GreenleafLab/ArchR#1651] [General]
Hi All,

We have cluster annotation from snRNAseq, while doing unconstrained integration of snATAC and snRNA seq, we lost clusters that was present in the snRNAseq seurat processing. Do we have to do both unconstrained and constrained integration for integration for having converage of all clusters in the snRNA seq. 

Thanks 
Divya

**Comment by rcorces** [comment]
> Do we have to do both unconstrained and constrained integration for integration for having coverage of all clusters in the snRNA seq.

No, you can use whichever one you want, or both depending on what makes the most sense for your data sets.
